test/integration/start_stop_delete_test.go

```diff
@@ -71,6 +71,9 @@ func TestStartStop(t *testing.T) {
 				"--disable-driver-mounts",
 				"--extra-config=kubeadm.ignore-preflight-errors=SystemVerification",
 			}},
+			{"docker", constants.DefaultKubernetesVersion, []string{
```

Comment:
We won't need to add this additional test. We can remove this

Reply:
done